File: Python/Convert_string_to_camel_case.py
def to_camel_case(text):
    s = text.replace('_','-')
    words_list = s.split('-')
    capitalized_list = [words_list[0]]
    for each_word in words_list[1:]:
        capitalized_list.append(each_word.title());
    return "".join(capitalized_list)
    
    

File: Javascript/ConvertStringToCamelCase.js
function toCamelCase(str) {
  let allwords = str.replace(/_/g,'-');
  allwords = allwords.split('-',);
  console.log(allwords)
  for (let i = 1; i < allwords.length; i++){
    allwords[i] = allwords[i].charAt(0).toUpperCase() + allwords[i].slice(1);
    
    
  }
  return(allwords.join(''))
}